.github/workflows/build.yml

```diff
@@ -18,6 +18,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Get yarn cache directory path
```

Comment:
Gonna check, thanks 👍

Reply:
Hm, it's unclear what the actual benefits of changing the cache  implementation would be. The official cache action Readme [recommends](https://github.com/actions/cache/blob/main/examples.md#node---yarn) the way it was implemented here. It seems to cut the install time by about 50% in our case, so that's quite good already. Will leave it as is for now.